internal/pkg/spinner/.travis.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
 language: go
```

Comment:
Thanks good catch! There are probably others 🧐

Reply:
Probably also `ui/lib/api-common-protos`, `ui/lib/grpc-web`, and `opaqueany` things.

Reply:
Although we do own `https://github.com/hashicorp/opaqueany` so maybe that's fine.

Reply:
Oh nice, the tool was smart enough to ignore `internal/pkg/jsonpb` because it already has a copyright in it. I'll list is as an exclusion though for posterity.